File: Attend-In App/app/src/main/java/com/example/attend_in/MainActivity.java
package com.example.attend_in;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageButton;

public class MainActivity extends AppCompatActivity {
    ImageButton attendImageButton;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        attendImageButton = (ImageButton) findViewById(R.id.attendImage);


        attendImageButton.setOnClickListener(new View.OnClickListener() {

            @Override
            public void onClick(View v) {
                Intent startIntent = new Intent(getApplicationContext(), SecondActivity.class); // so it can go to the second activity screen once you click on the Attend in button
                startActivity(startIntent);

            }
        });
    }
}

File: MainActivity.kt
package com.example.tutorialpart1

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*
import kotlinx.android.synthetic.main.content_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        setSupportActionBar(toolbar)

        studentButton.setOnClickListener{
            startActivity(Intent(this, StudentPath::class.java))

        teacherButton.setOnClickListener{
            startActivity(Intent(this, TeacherPath::class.java))
        }


    }
}
}